File: Plugins/Aspose_Cells_Java_for_Ruby/lib/asposecellsjava/displayhidescrollbars.rb
module Asposecellsjava
  module DisplayHideScrollBars
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(__FILE__))) + '/data/'
        
        # Instantiating a Workbook object by excel file path
        workbook = Rjb::import('com.aspose.cells.Workbook').new(data_dir + 'Book1.xls')
        
        # Hiding the vertical scroll bar of the Excel file
        workbook.getSettings().setVScrollBarVisible(false)

        # Hiding the horizontal scroll bar of the Excel file
        workbook.getSettings().setHScrollBarVisible(false)
        
        # Saving the modified Excel file in default (that is Excel 2003) format
        workbook.save(data_dir + "output.xls")

        puts "Scroll Bars are now hidden, please check the output file."
    end
  end
end

File: Plugins/Aspose_Cells_Java_for_PHP/src/aspose/cells/WorkingWithWorksheets/DisplayFeatures/DisplayHideScrollBars.php
<?php
/**
 * Created by PhpStorm.
 * User: fahadadeel
 * Date: 05/08/15
 * Time: 12:17 PM
 */

namespace Aspose\Cells\WorkingWithWorksheets\DisplayFeatures;

use com\aspose\cells\Workbook as Workbook;

class DisplayHideScrollBars {

    public static function run($dataDir=null)
    {

        //Instantiating a Excel object by excel file path
        $workbook = new Workbook($dataDir . "book1.xls");

        //Hiding the vertical scroll bar of the Excel file
        $workbook->getSettings()->setVScrollBarVisible(false);

        //Hiding the horizontal scroll bar of the Excel file
        $workbook->getSettings()->setHScrollBarVisible(false);

        //Saving the modified Excel file in default (that is Excel 2003) format
        $workbook->save($dataDir . "output.xls");

        // Print message
        print "Scroll bars are now hidden, please check the output document.";

    }

}